recipes-graphics/spark/spark_git.bb

```diff
@@ -0,0 +1,100 @@
+include pxcore.inc
+
+DEPENDS_append = " curl freetype util-linux libjpeg-turbo libpng pxcore-libnode giflib sqlite3 wpeframework-tools-native wpeframework wpeframework-clientlibraries"
+
+SRC_URI += "file://Spark.pc \
+    file://0001-nanosvg-patches.patch \
+    file://0002-pxScene-disable-2DMultisampleEXT.patch \
+    file://0003-spark-wpeframework-compositor.patch \
+    file://0004-dukluv-git.patch \
+    file://0005-pxScene-essos-support-for-shared-lib.patch \
+    file://0006-pxScene-initApp-load-issue-fix.patch \
+    file://0007-node-v10.15.3_mods.patch \
+"
+
+inherit cmake pkgconfig
+
+PACKAGECONFIG ?= " \
+    ${@bb.utils.contains('DISTRO_FEATURES', 'wayland', 'wayland westeros', 'wpeframework', d)} \
+"
+
+PACKAGECONFIG[rtremote]     = "-DBUILD_RTCORE_LIBS=ON -DBUILD_RTCORE_STATIC_LIB=OFF,,,rtcore rtremote"
```

Comment:
Ca you please remove additional whitespaces?

Reply:
done